File: argumentos.cpp
#include <iostream>
using namespace std;
int main(int argc, char *argv[])
{
    cout << "cantidad de argumentos: " << argc <<"\n[";
    for (int i = 0; i < argc; i++)
    {
        cout << argv[i] << ",";
    }
    cout << "]\n";
    return 0;
}

File: Argumentos.java
/**
 * Argumentos
 */
public class Argumentos {
    public static void main(String[] args) {
        System.out.print(args.length+"\n[");
        for (int i = 0; i < args.length; i++) {
            System.out.print(args[i]+","); 
        }
        System.err.println("]");
    }
}